🐛 Bug Report: npm ci error `ENOENT no such file or directory`

### 📜 Description

GitHub Action CI failing when running `npm ci`

[CI #2680](https://github.com/amplication/amplication/runs/4635650792)

Error:
Run npm ci
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /home/runner/work/amplication/amplication/node_modules/camel-case/bin/escodegen.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/runner/work/amplication/amplication/node_modules/camel-case/bin/escodegen.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-12-26T10_08_39_157Z-debug.log
Error: Process completed with exit code 254.

### 👟 Reproduction steps

Run CI build on
branch: release/0.11.0 
commit: [9395ecaa92f70b7addded9e24a6b6f89416f23ee](https://github.com/amplication/amplication/commit/9395ecaa92f70b7addded9e24a6b6f89416f23ee)

### 👍 Expected behavior

`npm ci` should pass successfully 

### 👎 Actual Behavior

`npm ci` failing

### 💻 Device

Desktop

### 💻 Operating system

Linux

### 🌍 Browser

Something Else

### 🧱 Your Environment

_No response_

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/amplication/amplication/blob/master/code_of_conduct.md)

--- a/package.json
+++ b/package.json
@@ -26,8 +26,8 @@
     "winston": "3.3.3"
   },
   "devDependencies": {
-    "@graphql-codegen/cli": "1.21.6",
-    "@graphql-codegen/typescript": "1.22.4",
+    "@graphql-codegen/cli": "1.21.8",
+    "@graphql-codegen/typescript": "1.23.0",
     "@types/jest": "27.0.2",
     "@types/semver": "7.3.8",
     "graphql": "15.3.0",

